include/lib/graft/task.h

```diff
@@ -193,8 +193,10 @@ class ExpiringList;
 class TaskManager : private HandlerAPI
 {
 public:
-    TaskManager(const ConfigOpts& copts);
+    TaskManager(const ConfigOpts& copts, SysInfoCounter* sysInfoCounter);
```

Comment:
const SysInfoCounter* ?

Reply:
No, we need the pointer to modify the counters.